cuml/dbscan_demo.ipynb

```diff
@@ -106,9 +120,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 9,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "output_type": "error",
```

Comment:
This seems to have some output in it. Would be nice to check in clean.

Reply:
Good catch! I may not have hit save when I cleared the output in Jupyter. Fixed.